dali/base/dafdesc.cpp

```diff
@@ -3316,7 +3324,7 @@ static void generateHosts(IPropertyTree * storage, GroupInfoArray & groups)
             cur.container = nullptr;
         }
 
-        if (cur.groupType == grp_thor)
+        if (cur.groupType == grp_thor && !cur.container)
```

Comment:
I think could benefit from a comment here to explain that the sort order has ensured will see thor 1st and hence container null , even if other groups that matched

Reply:
Added